docs/setup/hooks.md

```diff
@@ -116,7 +116,7 @@ For example, lakeFS will search and execute all matching `Action` files with the
 
 ## Runs API & CLI
 
-[OpenAPI](../reference/api.md) endpoint and [lakectl](../reference/commands.md#lakectl-actions) expose the results of `Runs` execution per repository, branch, commit and specific `Action`.
+[LakeFS API](../reference/api.md) endpoint and [lakectl](../reference/commands.md#lakectl-actions) expose the results of `Runs` execution per repository, branch, commit and specific `Action`.
```

Comment:
```suggestion
The [lakeFS API](../reference/api.md) and [lakectl](../reference/commands.md#lakectl-actions) expose the results of executions per repository, branch, commit and specific `Action`.
```

(today, we only write lakeFS with a lowercase r. This is an ongoing debate but for now let's keep it this way).